packages/confirmation-modal/src/ConfirmationModal.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@leafygreen-ui/emotion';
+import Button, { Variant } from '@leafygreen-ui/button';
+import Modal from '@leafygreen-ui/modal';
+import { uiColors } from '@leafygreen-ui/palette';
+import TextInput from '@leafygreen-ui/text-input';
+
+const titleStyle = css`
+  color: ${uiColors.gray.dark2};
+  font-size: 24px;
+  font-weight: bold;
+  line-height: 25px;
+
+  margin-bottom: 10px;
+`;
+
+const baseModalStyle = css`
+  width: 600px;
+  padding: initial;
+`;
+
+const contentStyle = css`
+  color: ${uiColors.gray.dark1};
+  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
+  font-size: 16px;
+  letter-spacing: 0;
+  line-height: 24px;
+
+  padding: 36px;
+`;
+
+const textEntryInputStyle = css`
+  width: 300px;
+  padding-top: 14px;
+`;
+
+const footerStyle = css`
+  &:before {
+    content: '';
+    display: block;
+    position: relative;
+    width: 100%;
+    border-top: 1px solid ${uiColors.gray.light2};
+  }
+`;
+
+const footerContentStyle = css`
+  display: flex;
+  justify-content: right;
+  flex-direction: row-reverse;
+  column-gap: 5px;
```

Comment:
I believe this is a property that works with CSS grid, but not flexbox. Right now, there's no margin between the two buttons in the footer.

Reply:
I think you're thinking of `grid-column-gap` which is being deprecated in favor of `column-gap`, which supports more contexts such as flex https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap#Examples

Reply:
Ahh you're right. Regardless, this implementation seems to be broken from my testing locally. Margin seems like a cleaner solution in this case anyways.

Reply:
Yeah interesting, interesting case of IWOMM. I'll switch to margin.